compose.yml

```diff
@@ -31,10 +34,11 @@ services:
       - LIBP2P_REPLICAS=${DOCKER_COMPOSE_LIBP2P_REPLICAS:-1}
       - ETCDCTL_ENDPOINTS=${DOCKER_COMPOSE_ETCDCTL_ENDPOINTS:-etcd:2379}
       - ETCDCTL_API=${DOCKER_COMPOSE_ETCDCTL_API:-3}
-      - NODE_MASK=${DOCKER_COMPOSE_LIBP2P_NODE_KEY_MASK:-2000000000000000000000000000000000000000000000000000000000000000}
+      - BOOTSTRAP_NODE_KEY=${DOCKER_COMPOSE_BOOSTRAP_NET_NODE_KEY:-1000000000000000000000000000000000000000000000000000000000000000}
```

Comment:
It seems that this `BOOTSTRAP_NODE_KEY` is not used by the `run_nomos_node.sh`and any sub-scripts.

Reply:
Im curious, why is it negative?

Reply:
This is a notation for a default value if the one from `.env` is absents. Agree that it's quite misleading on first glance.

Reply:
> It seems that this BOOTSTRAP_NODE_KEY is not used by the run_nomos_node.shand any sub-scripts.

Thanks for spotting that, fixed.